ACE/ace/OS_NS_Thread.cpp

```diff
@@ -1151,11 +1151,11 @@ ACE_OS::cond_broadcast (ACE_cond_t *cv)
         result = -1;
       // Wait for all the awakened threads to acquire their part of
       // the counting semaphore.
-#   if defined (ACE_VXWORKS) || defined (ACE_MQX)
+#   if defined (ACE_VXWORKS) || defined (ACE_MQX) || defined (INTEGRITY)
```

Comment:
What about adding ACE_INTEGRITY instead of INTEGRITY so that the define style is consistent